### ISSUE TITLE
fails silently if launchConfig does not contain files value

### ISSUE THREAD

**Issue by georgejecook** [TwitchBronBron/vscode-brightscript-language#174]
if files value is not present in a launchConfig, then the adapter will experience a crash, as in this location, due to files being undefined:

```RokuDeploy.prototype.normalizeFilesOption = function (files) {
        var result = [];
        var topLevelGlobs = [];
        //standardize the files object
        for (var _i = 0, _a = files; _i < _a.length; _i++) {```

**Comment by chrisdp** [comment]
Opened a pr that should address this #180

**Comment by TwitchBronBron** [comment]
This was fixed as a result of #180.
